lib/journaled/stream_name.rb

```diff
@@ -0,0 +1,6 @@
+module Journaled
+  module_function def stream_name_for_app(app_name)
+    env_var_name = [app_name&.upcase, 'JOURNALED_STREAM_NAME'].compact.join('_')
```

Comment:
Would app_name ever be nil? Just thinking if the safety operator is really needed

Reply:
Yup! In the `default` case `app_name` is gonna be nil

Reply:
Lol, I guess the spec below answered that question 😄